### ISSUE TITLE
Chronos: Embrace bigdl-nano in Chronos

### ISSUE THREAD

**Issue by TheaperDeng** [intel-analytics/BigDL#3171]
### Background

Currently `Chronos` rely on `orca.automl` for their pytorch model single node support. This will cause

- Future `chronos-inference` will rely on `orca` (and `dllib`) which make the whole package fat.
- Can not use `nano`'s speed up.
- pytorch-lightning Trainer will bring more feature to Chronos (such as efficient lr/bs tuning)

### Implementation Step

We will implement the whole plan mainly in 5 PRs

- [x] add a new `nano.pytorch.pl_base_model`, this model should add time series specific training step ...

- [x] add `chronos.metric` to replace `orca.automl.metric`. #3608 

- [x] change forecaster(distributed=False) path to bigdl-nano(pl) #3539 

- [x] change TSPipeline path to bigdl-nano(pl) #3609 

- [x] change Automodel (except .fit) to bigdl-nano(pl) #3745

**Comment by TheaperDeng** [comment]
> 1. **distributed=false**: using `nano` (possibly with pseudo-distributed mode)
> 2. **distributed=true**: using `orca`

yes, that is the plan
